packages/babel-core/test/async.js

```diff
@@ -9,15 +9,14 @@ import {
   supportsESM,
 } from "./helpers/esm.js";
 
-import { itGte, itESM, itLt } from "$repo-utils";
+import { itGte, itESM, itNegate, itSatisfies } from "$repo-utils";
 
 // "minNodeVersion": "8.0.0" <-- For Ctrl+F when dropping node 6
 const nodeGte8 = itGte("8.0.0");
 const nodeGte14 = itGte("14.8.0");
 
 // "minNodeVersion": "22.0.0" <-- For Ctrl+F when dropping node 20
-const nodeGte22_12 = itGte("22.12.0");
-const nodeLt22_12 = itLt("22.12.0");
+const nodeGte22_12OrHat20_19 = itSatisfies("^20.19.0 || >= 22.12.0");
```

Comment:
We could also use `process.features.require_module` instead of version checks

Reply:
Also, could we call it `itSupportsRequireESM` instead of `nodeGte22_12OrHat20_19`, which is otherwise a pain to read? :P

Reply:
> process.features.require_module

It seems that node.js may deprecate certain features when it has been always `true` for a long while, such as `ipv6` deprecated in v23.4.0. Let's say if node.js 30 deprecates `require_module` but Babel 8 still has to support node.js 20, then we will see deprecated warning when running test on node.js 30 while the check is essential for node.js 20. So I prefer the `version` check since it is safe from future deprecation.